speechmatics/cli_parser.py

```diff
@@ -283,6 +283,24 @@ def get_arg_parser():
         help="Perform sentiment analysis on the transcript",
     )
 
+    # Parent parser for batch topic-detection argument
+    batch_topic_detection_parser = argparse.ArgumentParser(add_help=False)
+    batch_topic_detection_parser.add_argument(
```

Comment:
for a CLI flag `speechmatics transcribe --detect-topics` reads nicer than `--topic-detection`. It helps the user know that it's something on the transcription IMO

Reply:
agree. it's just that --sentiment-analysis seems a bit out of place now, I wish it was standardized. But at least we follow the --summarize flag